api-spec.md

```diff
@@ -30,20 +30,20 @@ Login
 Reporting
 ---------
 
-#### Report Diagnosis: `POST /reports/`
+#### Report Diagnosis: `POST /cenreport/`
```

Comment:
Since they renamed CEN to TCN (AFAIK just yesterday), I think we should follow their naming convention for clarity and consistency, so `POST /tcnreport` or `POST /tcn_report` seems more appropriate to avoid confusion in the future.